components/json/rs/src/lib.rs

```diff
@@ -0,0 +1,121 @@
+#[cxx::bridge(namespace = json)]
+mod ffi {
+    extern "Rust" {
+        fn convert_uint64_value_to_string(path: &str, json: &str) -> String;
+        fn convert_int64_value_to_string(path: &str, json: &str) -> String;
+        fn convert_string_value_to_uint64(path: &str, json: &str) -> String;
+        fn convert_string_value_to_int64(path: &str, json: &str) -> String;
+    }
+}
+
+// Parses and re-serializes json with the value at path converted from a uint64
+// to a string representation of the same number.
+// Returns an empty String if such conversion is not possible.
+// A path is a Unicode string with the reference tokens separated by /.
```

Comment:
can we use `.` instead of `/` to match upstream (prefs, etc..)? Also a key could potentially contain the separator so we need to handle escaping

Reply:
seems no, https://docs.serde.rs/src/serde_json/value/mod.rs.html#816-847